scipy/linalg/special_matrices.py

```diff
@@ -469,6 +469,51 @@ def kron(a, b):
     return np.concatenate(np.concatenate(o, axis=1), axis=1)
 
 
+def Khatri_rao(a, b):
+    """
+    Khatri-rao product
+
+    The result is the block matrix.
+
+    Mathematically:
+    ---------------
+    A * B = (A_{ij}  \bigotimes B_{ij})_{ij}
+
+    Parameters
+    ----------
+    a:  (I, K) ndarray
+        Input array
+    b:  (J, K) ndarray
+        Input array
+
+    Returns
+    -------
+    C:  ((I*J), K) ndarray
```

Comment:
Better to use `ndarray` here, and put `((I*J), K)` (surrounded by double backticks for correct typesetting) in the description on the next line.

One thing that's otherwise going wrong here is that `*` is a special symbol in reST formatting, and will mess things up

Reply:
done